app/Path.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
```

Comment:
You can add docblocks here for mapping fields from Eloquent. It will be helpful later (of course if you're using smart IDE):
```
/**
 * @property int $id
 * (...)
 */
class Path extends Model
```